sonar-text-dotnet/src/SonarLint.Secrets.DotNet.UnitTests/DotNetOnly/RegexMatcherTest_NoJavaTests.cs

```diff
@@ -35,7 +35,7 @@ public void Find_Simple()
             actual.First().Length.Should().Be(3);
             actual.First().Text.Should().Be("123");
 
-            actual.Last().StartIndex.Should().Be(52);
+            actual.Last().StartIndex.Should().Be(50);
```

Comment:
That's an interesting change in results. Do you know what change caused the difference?

Reply:
From some tests I did I assume it is the line breaks.
My IDE and the pipeline seem to work with LF (because the tests were failing also in cirrus).
If I remove the line breaks it correctly figures out the starting point of both passwords - when I add the line breaks back and I count them as one character (LF) then the actual start index of the second password is 50. 
I guess with CRLF (2 chars) is +2.